build/helper/helper.py

```diff
@@ -56,7 +56,10 @@ def get_python_type_for_visa_type(visa_type):
         'ViRsrc': 'str',
     }
 
-    return type_map[visa_type]
+    if visa_type in type_map:
+        return type_map[visa_type]
+    else:
+        return 'int'
```

Comment:
When did you encounter this?
This catch-all makes me a little nervous.

Reply:
FGEN has NIComplexNumber and NIComplexI16. I was not planning on supporting custom (non-visa) types in this PR.

Reply:
Can you return None or '' or something obviously invalid?
We should not be generating code for these functions yet anyway.

Reply:
Not without putting workarounds elsewhere that I don't want to do just to make this work.

This way, all the templates generate SOMETHING. That something may not be correct for the DLL though. That will need custom types (or some other mechanism)

Reply:
> That something may not be correct for the DLL though. 

This is what worries me, silently generating code that calls into the DLL incorrectly, corrupting memory or crashing. If we simply assert "type not supported" but _only_ when the function is to be generated, that may be enough. Workaround is to temporarily disable code generation.